modules/oxide/src/less/theme/components/spacer/spacer.less

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright (c) Tiny Technologies, Inc. All rights reserved.
+// Licensed under the LGPL or a commercial license.
+// For LGPL see License.txt in the project root for license information.
+// For commercial licenses see https://www.tiny.cloud/
+//
+
+.tox {
+  .tox-spacer {
```

Comment:
For maintenance sake, we need to document what a spacer is and what it is used for

Reply:
The same could be said of many other components. It will be added to the editor documentation, is that enough?

Reply:
Spacer has been done another way.